### DISCUSSION TITLE
Keyboard controls for Contrast

### DISCUSSION THREAD

**Discussion by Muirium** [MonitorControl/MonitorControl#603] [Ideas]
Hi folks: here's a little feature request, hopefully simple to do!

I'd love to get direct keyboard controls for Contrast as well as Brightness. My Dell P2415Q is old-school, that way: both need tweaked a little differently. ([Gory details here](https://github.com/MonitorControl/MonitorControl/discussions/596#discussioncomment-1333834).)

For now, I have to tweak Contrast with the mouse, right here:
![](https://user-images.githubusercontent.com/221646/133448363-c59fa9c4-a46f-4910-b45c-8104fa613ef3.jpg)
MonitorControl gives me nice direct control over both Brightness and Contrast levels this way. Phew! I simply want to access both of them from the keyboard, which is the fastest way for ++++ / -- tweaks!

I'm thinking F1 & F2 for Brightness control and Shift+F1 & Shift+F2 for Contrast control. (I use vintage mechanical keyboards with classic Function rows.) Naturally, configurable keys, just like Brightness already has, are even better!

Some displays really need their Contrast tweaked separately. I'd love to make it as effortless as Brightness control already is in this most excellent app!

#596

**Comment by waydabber** [comment]
This was added as a Feature Request, see #621

**Reply by waydabber** [reply]
This is now completed, see https://github.com/MonitorControl/MonitorControl/issues/621
